SideBarFolders.py

```diff
@@ -342,6 +342,51 @@ def run(self):
 		s.set("swap_append_load", not current_swap)
 		sublime.save_settings('Side Bar Folders.sublime-settings')
 
+class side_bar_folders_quick_switch(sublime_plugin.WindowCommand):
+	def run(self):
+		folder_list = list(map(lambda folder: folder['path'], s.get('folders', [])))
+		folder_list.reverse()
+		self.display_list(folder_list)
+
+	def display_list(self, list):
+		self.window.show_quick_panel(list, self.on_done)
+
+	def on_done(self, index = -1, append = False):
+		print(index)
+		folder = (Pref.folders[::-1])[index]
+		if self.audit_folder(folder, index):
+			return
+		project = get_project_data(Window())
```

Comment:
In some situations that I cannot remember window and view are incorrect, so I always use sublime.active_window() and sublime.active_view() which I can certainly be sure that the value is correct.